src/common/components/Field/Field.tsx

```diff
@@ -25,7 +27,10 @@ export const Field = ({
   const { wrapper } = fieldTheme();
   return (
     <div {...props} className={wrapper()}>
-      {label && <Label text={label} size={size} />}
+      <div className="flex flex-wrap justify-between">
```

Comment:
Can you add a small gap here? So if contents were to be very close, they won't be too close.

Lets try `gap-2` for now!

Reply:
added at fdf32ba!